style/pi-hole.css

```diff
@@ -1582,3 +1582,9 @@ textarea.field-sizing-content {
   line-height: 0.5em;
   font-size: 1.7em;
 }
+
+#logout-button {
+  color: inherit;
+  padding: 6px 0;
+  text-decoration: none;
+}
```

Comment:
There is a small border of 2px, causing a small spacing difference.

Also, all other links take up 100% of the width. 
This is not normally visible because the links and this button have transparent background, but browsers forcing "focus-visible" for accessibility, this button will look shorter than the links above. 

```suggestion
#logout-button {
  color: inherit;
  padding: 6px 0;
  text-decoration: none;
  width: 100%;
  text-align: inherit;
  border: none;
}
```

Reply:
I don't see this, please share a screenshot. The default outline is nice as it is and good for a11y.

Reply:
![image](https://github.com/user-attachments/assets/b93c871e-554e-4762-97a7-d71b529f250f)

Reply:
Your own screenshot shows the issue.
Compare the last button with the other links:
![image](https://github.com/user-attachments/assets/16d99edf-ab33-43bc-afdd-1ca794aa9722)


The Logout button is shorter than the others.

Now using my suggested CSS:

![image](https://github.com/user-attachments/assets/e8a6e0b2-6aa3-4c8f-bfe1-bac0635c70d2)